app/views/hyrax/base/_form_media.html.erb

```diff
@@ -1,4 +1,8 @@
 <% if f.object.persisted? && f.object.member_ids.present? %>
   <%= render 'form_representative', f: f %>
   <%= render 'form_thumbnail', f: f %>
+  <%# display the rendering dropdown only if the rendering responds to image? (ie. Universal Viewer is used) %>
+  <% if curation_concern.representative.present? && curation_concern.representative.image? %>
```

Comment:
Why do we have this conditional?  Shouldn't we be able to select the representative and the rendering all in one step?  Also I'm not sure that responding to `image?` implies that UV is being used (which is how I read this comment)

Reply:
Assigning rendering if there isn't an image assigned to representative will have no effect. It won't even appear in the manifest as there's no sequences block to add it to. That's why I originally did it conditionally, to avoid confusion of having a pointless option. But the downside is that it can take two edits to 1) assign representative, 2) assign rendering. I've removed the conditional.